packages/elements-core/src/components/TryIt/build-request.ts

```diff
@@ -50,6 +50,68 @@ const getServerUrl = ({
   return serverUrl;
 };
 
+const getQueryParams = ({
```

Comment:
Would you mind adding some tests specifically for this function?
The table you added is great, but I'm afraid further contributors may potentially forget about it in case a change is needed.

LMK if you have any questions.

Reply:
Sure thing! I was trying to match the code style around and noticed you don't write unit tests for individual files and functions. Having unit tests here would be lovely as there is a clear specification that the function needs to conform to.

Reply:
Done. Let me please know if this is what you were expecting. I've checked coverage and it's basically 100% covered. I've noticed you don't use `test:` prefix for commits so I've used `chore:` here, but I guess you'll squash my commits anyway.

Reply:
Yeah, we'll squash the commits anyway, so it doesn't matter that much.

Thanks!